docassemble/AssemblyLine/al_courts.py

```diff
@@ -43,25 +44,41 @@ def short_label(self)->str:
       append city name to the court name. This is good for a drop-down selection
       list.
       """
-      if self.address.city in str(self.name):
-        return str(self.name)
+      # Avoid forcing the interview to define the court's address
+      if defined( self.attr_name('address.city') ):
+        if self.address.city in str(self.name):
+          return str(self.name)
+        else:
+          return str(self.name) + ' (' + self.address.city + ')'
       else:
-        return str(self.name) + ' (' + self.address.city + ')'
+        return str( self.name )
     
     def short_label_and_address(self)->str:
       """
       Returns a markdown formatted string with the name and address of the court.
       More concise version without description; suitable for a responsive case.
       """
-      return '**' + self.short_label() + '**' + '[BR]' + self.address.on_one_line()
+      label = f'**{ self.short_label() }**'
+      
```

Comment:
This behavior no longer matches the name of the method.

Reply:
This was meant to display information correctly if the user omits certain pieces of the address. Is that not desired? Do we expect that addresses will only be through developers?

Reply:
Let's make the method match its name. It would be confusing otherwise. Safety is a good goal but this is not an instance where it is needed.

This method would never be used in a user input situation. It is to aid with selecting the court from a list of predefined courts.